flight/Modules/Telemetry/telemetry.c

```diff
@@ -796,6 +796,7 @@ static int32_t setUpdatePeriod(telem_t telem, UAVObjHandle obj,
  */
 static void gcsTelemetryStatsUpdated(telem_t telem)
 {
+	// XXX this is dumb.
```

Comment:
Doing telemetry handshake in telemetry seems... not dumb? Can you explain this more precisely?

Reply:
Committed an extended comment :D